src/awkward/_pickle.py

```diff
@@ -72,11 +73,24 @@ def get_custom_reducer() -> PickleReducer | None:
             return _plugin
 
 
-def custom_reduce(obj, protocol) -> tuple | NotImplemented:
-    plugin = get_custom_reducer()
-    if plugin is None:
+_DISABLE_CUSTOM_REDUCER = False
+
+
+@contextmanager
+def use_builtin_reducer():
+    global _DISABLE_CUSTOM_REDUCER
+    old_value, _DISABLE_CUSTOM_REDUCER = _DISABLE_CUSTOM_REDUCER, True
+    try:
+        yield
+    finally:
+        _DISABLE_CUSTOM_REDUCER = old_value
+
+
+def custom_reduce(obj, protocol: int) -> tuple | NotImplemented:
+    if (plugin := get_custom_reducer()) is None or _DISABLE_CUSTOM_REDUCER:
         return NotImplemented
-    return plugin(obj, protocol)
+    else:
+        return plugin(obj, protocol)
```

Comment:
We want to be able to disable this custom reduction mechanism within our test suite. I think the easiest method is this one: temporarily disabling the custom reducer plugin.

Reply:
Okay!